ui/components/multichain-accounts/address-qr-code-modal/address-qr-code-modal.tsx

```diff
@@ -0,0 +1,249 @@
+                // Background and border must remain white regardless of theme
+                className="bg-white border-4 border-white rounded-2xl"
+              />
+
+              <Box
+                // Background and border must remain white regardless of theme
+                className="absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 border-8 border-white bg-white rounded-xl flex"
+                justifyContent={BoxJustifyContent.Center}
+                alignItems={BoxAlignItems.Center}
+              >
+                <AvatarNetwork
+                  name={networkName}
+                  src={networkImageSrc}
+                  className="bg-white" // stay white regardless of theme
+                />
+              </Box>
+            </Box>
+
+            <div>
+              <Text
+                textAlign={TextAlign.Center}
+                variant={TextVariant.HeadingSm}
+              >
+                {t('addressQrCodeModalHeading', [networkName])}
+              </Text>
+              <Text
+                textAlign={TextAlign.Center}
+                color={TextColor.TextAlternative}
+              >
+                {t('addressQrCodeModalDescription', [networkName])}
+              </Text>
+            </div>
+
+            <Box
+              flexDirection={BoxFlexDirection.Column}
+              justifyContent={BoxJustifyContent.Center}
+              alignItems={BoxAlignItems.Center}
+              gap={1}
+              className="w-full"
+            >
+              <Text
+                textAlign={TextAlign.Center}
+                variant={TextVariant.BodyMd}
+                fontWeight={FontWeight.Medium}
+                className="break-all max-w-64"
+              >
+                <Text asChild>
+                  <span>{addressStart}</span>
+                </Text>
+                <Text asChild color={TextColor.TextAlternative}>
+                  <span>{addressMiddle}</span>
+                </Text>
+                <Text asChild>
+                  <span>{addressEnd}</span>
+                </Text>
+              </Text>
+              <Button
+                variant={ButtonVariant.Tertiary}
+                endIconName={copied ? IconName.CopySuccess : IconName.Copy}
+                size={ButtonSize.Lg}
+                isFullWidth
+                onClick={handleCopyClick}
+              >
+                {t('copyAddressShort')}
+              </Button>
+            </Box>
+
+            {explorerInfo && (
+              <Button
+                variant={ButtonVariant.Secondary}
+                size={ButtonSize.Lg}
+                isFullWidth
+                onClick={handleExplorerNavigation}
+                asChild
+              >
+                <a
+                  href={`${explorerInfo.url}/address/${address}`}
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  {explorerInfo.buttonText}
+                </a>
```

Comment:
Rendering view explorer as a <a> tag which is more semantically correct and improves accessibility